.rebase/CHANGELOG.md

```
@@ -15,6 +16,16 @@ https://github.com/che-incubator/che-code/pull/720
 - code/remote/package.json
 ---
 
+#### @sbouchet
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)